inst/metadata/anc_run_options.json

```diff
@@ -3,19 +3,19 @@
   "description": "t_(OPTIONS_ANC_DESCRIPTION)",
   "controlGroups": [
     {
-      "label": "t_(OPTIONS_ANC_PREVALENCE_LABEL) - t_(OPTIONS_T1_LABEL)",
+      "label": "t_(OPTIONS_ANC_CLIENTS_LABEL)",
```

Comment:
This looks a bit weird to have a column where first entry is time 2 and then subsequent in column are both time 1. Though not sure how you would neaten it up as first row is required